lib/templates/swift3-variant/inc/_attributes_enum.liquid

```diff
@@ -2,7 +2,7 @@
 {% if entity.attributes.size > 0 %}
   enum Attributes: String {
     {%- for attribute in entity.attributes -%}
-        {%- if attribute.realm_ignored or attribute.realm_read_only.size == 0 -%}
+        {%- if attribute.realm_ignored == false -%}
```

Comment:
🤔  how come did it work before that `== false`?

Reply:
I don't know how it worked before I remove realmReadOnly, it was a bug. But with unit tests I saw that `attribute.realm_ignored` is not a valid statement. So I copy existing comparison and now it works with `attribute.realm_ignored == false`